selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/CodenvyEditor.java

```diff
@@ -141,8 +156,7 @@ public CodenvyEditor(
     String ORION_CONTENT_ACTIVE_EDITOR_XPATH = ORION_ACTIVE_EDITOR_CONTAINER_XPATH + "/div";
     String ACTIVE_LINES_XPATH =
         "//div[@class='textviewSelection']/preceding::div[@class='annotationLine currentLine'][1]";
-    String ACTIVE_LINE_HIGHLIGHT =
-        "//div[@class='annotationLine currentLine' and @role='presentation']";
+    String ACTIVE_LINE_HIGHLIGHT = "//div[@class='annotationLine currentLine']";
```

Comment:
XPath could be replaced by simpler variant - class name "`annotationLine currentLine`".

Reply:
This is constant is not being used now. But it can be useful in the future. If I change the current constant to class name "annotationLine currentLine" it will be hard to understand what it means. Because it looks like "Id". In my opinion, "xpath" is better than constant name "CLASS_NAME_ACTIVE_LINE_HIGHLIGHT" , or comment.

Reply:
XPath is the worst way to refer a web element so as it consists of redundant elements, syntax sugar etc, which make it much harder to read, understand and change. 
Just compare:
`String ACTIVE_LINE_HIGHLIGHT = "//div[@class='annotationLine currentLine']";`
with
`String ACTIVE_LINE_HIGHLIGHT_CLASS="annotationLine currentLine";`

Reply:
"ACTIVE_LINE_HIGHLIGHT_CLASS" - I am thinking about the highlighted class name or something else but not about "By.className" locator.

Reply:
let's name it "ACTIVE_LINE_HIGHLIGHT__CLASS"